layouts/partials/site-header.html

```diff
@@ -14,9 +14,50 @@
     {{ if .Site.Menus.main }}
       <div class="navbar-start">
         {{ range .Site.Menus.main }}
-          <a href="{{ .URL }}" class='navbar-item has-text-centered {{ cond (eq $.RelPermalink .URL) " is-active" "" }}'>
-            {{ cond (eq .Name "Github") (partial "github-logo" .)  .Name }}
-          </a>
+          {{ if (eq .Name "Resources") }}
+            <div class="navbar-item has-text-centered has-dropdown is-hoverable">
+              <a href="{{ .URL }}" class="navbar-link">{{ .Name }}</a>
+              <div class="navbar-dropdown">
+                <a class="navbar-item" href="{{ relURL "newsletter" }}">
+                  Newsletter
+                </a>
+                <a class="navbar-item" href="/#getting_started">
+                  Getting Started
+                </a>
+                <a class="navbar-item" href="{{ relURL "resources/#books" }}">
+                  Books
+                </a>
+                <a class="navbar-item" href="{{ relURL "videos" }}">
+                  Videos
+                </a>
+                <a class="navbar-item" href="{{ relURL "related_projects" }}">
+                  Releated Projects
+                </a>
+                <a class="navbar-item" href="{{ relURL "trademarks" }}">
+                  Trademarks Policy
+                </a>
+                <p class="menu-label">
+                  External links
```

Comment:
Maybe we can place an "external" icon next to external links. 
https://www.google.com/search?q=external+link+icon&tbm=isch

Reply:
That's a good idea. I'll adjust